slave-maven/Dockerfile

```diff
@@ -2,12 +2,12 @@ FROM openshift/jenkins-slave-base-centos7
 
 MAINTAINER Ben Parees <bparees@redhat.com>
 
-ENV MAVEN_VERSION=3.3 \
-    GRADLE_VERSION=3.5 \
+ENV MAVEN_VERSION=3.5.0 \
```

Comment:
same here.  this env var doesn't control the version that's installed, it's chosen by the package name below, but there is no 3.5 package available yet.